src/main/java/uk/gov/hmcts/reform/hmc/model/PartyDetails.java

```diff
@@ -26,7 +26,7 @@ public class PartyDetails {
     private String partyType;
 
     @NotEmpty(message = ValidationError.PARTY_ROLE_EMPTY)
-    @Size(max = 6, message = ValidationError.PARTY_ROLE_MAX_LENGTH)
+    @Size(max = 40, message = ValidationError.PARTY_ROLE_MAX_LENGTH)
```

Comment:
You will need to change the value of `ValidationError.PARTY_ROLE_MAX_LENGTH` from

`public static final String PARTY_ROLE_MAX_LENGTH = "Party role must not be more than 6 " + CHARACTERS_LONG;`

to

`public static final String PARTY_ROLE_MAX_LENGTH = "Party role must not be more than 40 " + CHARACTERS_LONG;`

Reply:
done